Shaders/TiltShift.fx

```diff
@@ -1,47 +1,56 @@
-// Based on kingeric1992's TiltShift effect
+/* 
+Tilt-Shift PS (c) 2018 Jacob Maximilian Fober, 
+(based on TiltShift effect (c) 2016 kingeric1992)
+
+This work is licensed under the Creative Commons 
+Attribution-ShareAlike 4.0 International License. 
+To view a copy of this license, visit 
+http://creativecommons.org/licenses/by-sa/4.0/.
+*/
 
 uniform bool Line <
 	ui_label = "Show Center Line";
 > = false;
 
-uniform float Axis <
+uniform int Axis <
```

Comment:
Why is this an `int` now?

Reply:
Yes, since its in degrees its easier to just drag. 1 degree seems to be fine precision.